Connection-Issues.md

```diff
@@ -0,0 +1,27 @@
+# Connection Issues
+
+In rare occasions, Internet Service Providers have issues on their end preventing users from connecting to [Runelite's website](https://runelite.net/), which is a requirement for starting Runelite. While this problem is not the fault of the user, it can often be mitigated with one of the workarounds listed below.
```

Comment:
branding capitalization
```suggestion
In rare occasions, Internet Service Providers have issues on their end preventing users from connecting to [RuneLite's website](https://runelite.net/), which is a requirement for starting RuneLite. While this problem is not the fault of the user, it can often be mitigated with one of the workarounds listed below.
```